editoast/src/redis_utils.rs

```diff
@@ -46,6 +49,30 @@ impl ConnectionLike for RedisConnection {
     }
 }
 
+impl RedisConnection {
+    pub async fn json_get<T: DeserializeOwned, K: ToRedisArgs + Send + Sync>(
+        &mut self,
+        key: K,
+    ) -> Result<Option<T>> {
+        let value: Option<String> = self.get(key).await?;
+        match value {
+            Some(v) => Ok(serde_json::from_str(&v)?),
+            None => Ok(None),
+        }
+    }
+
+    pub async fn json_set_ex<K: ToRedisArgs + Send + Sync, T: Serialize>(
+        &mut self,
+        key: K,
+        value: &T,
+        seconds: u64,
+    ) -> Result<()> {
+        let str_value = serde_json::to_string(value)?;
```

Comment:
We should transform the SerdeError into a Redis custom error (we can type it as Serialize). This will throw a 500 instead of a 400.